src/remote.js

```diff
@@ -3,6 +3,7 @@
 import {Node, Commands, Constants, MessagesQueue, StyleAttributes, EventDOMNodeAttributes, SupportedEvents} from './common';
 
 let index = 0;
+let containerPromiseResolver;
```

Comment:
This pattern does not allow multiple "createContainer" invocations.
I think it'll be better to change it to something like the following.

let initMsgPromiseResolver;
const initMsgPromise = new Promise(resolve => {initMsgPromiseResolver = resolve});

Then, in createContainer:
return initMsgPromise.then(() => res);

And in handleMessageFromPipe:
initMsgPromiseResolver();